autoloads/local_high_scores.gd

```diff
@@ -24,13 +24,7 @@ func load():
 		file.close()
 
 		if typeof(data) == TYPE_ARRAY:
-			# for backward compat, only needed for cam & tris once.
-			# Replace if/else with just scores=data afterward
-			if data.size() > 0 && data[0].has("Name"):
-				scores = _convert_score_file(data)
-				save()
-			else:
-				scores = data
```

Comment:
@avelitj12 am I right in assuming nobody else has played the game locally? and that you've converted your local file?

Reply:
I can delete the local save files on mine and Chyanne's computers